rush/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -181,6 +189,39 @@ export default class RushConfiguration {
     return tempNamesByProject;
   }
 
+  private static _validateCommonRushConfigFolder(commonRushConfigFolder: string): void {
```

Comment:
>_validateCommonRushConfigFolder [](start = 17, length = 31)

So this is basically just making sure no known files are in the rush config folder? Can you add a comment mentioning that?